alfred-api-integrationtests-client/alfresco/src/main/java/eu/xenit/alfred/api/tests/search/SearchServiceTest.java

```diff
@@ -378,7 +375,7 @@ public void invalidLong_onlyNode_doesNotThrowException() {
         SearchSyntaxNode node =
                 new QueryBuilder()
                         .startOr()
-                        .property("{http://www.alfresco.org/model/system/1.0}node-dbid", LONG_MAX_VALUE_PLUS_ONE)
+                        .property("{http://www.alfresco.org/model/system/1.0}node-dbid", LONG_MAX_VALUE)
```

Comment:
This seems like something that was built to overflow the max_long, now it's not used? Or the overflow already occurs on LONG_MAX_VAL?